CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.22.0 (TBD)
+## 1.22.0 (2024-09-10)
 
 ### Snowpark Python API Updates
 
```

Comment:
On line 9, add "the" like this:

Added the following new functions in `snowflake.snowpark.functions`:

Reply:
On line 57, the sentence is missing a period at the end.

Reply:
updated